lib/src/delegates/assets_picker_text_delegate.dart

```diff
@@ -281,3 +281,48 @@ class RussianTextDelegate implements AssetsPickerTextDelegate {
   String durationIndicatorBuilder(Duration duration) =>
       AssetsPickerTextDelegate.defaultDurationIndicatorBuilder(duration);
 }
+
+/// [AssetsPickerTextDelegate] implements with Russian.
+/// Локализация на русский язык.
```

Comment:
🤔

Reply:
Thank you for checking and I'm sorry for my mistake.
I revised and push it again.